ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -101,9 +106,28 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'R2S Headmarker Spread',
       type: 'HeadMarker',
-      netRegex: { id: headMarkerData.spreadMarker2, capture: false },
-      suppressSeconds: 5,
-      response: Responses.spread(),
+      netRegex: { id: headMarkerData.spreadMarker2, capture: true },
+      delaySeconds: (data, matches) => {
+        data.beatTwoSpreadCollect.push(matches.target);
+        return data.beatTwoSpreadCollect.length === 2 ? 0 : 0.1;
+      },
```

Comment:
(Opinion only, this is non-binding.)

This would avoid implementing a second trigger for collecting the spreads, but I'm not sure we want to start doing this. I do think there's nebulous value in having dedicated collection triggers, and stacking it all into one trigger ends up making it large vertically. Is there a specific reason to combine it all, or is this just "oh, hey, we can do this thing"?

Reply:
I just followed the suggestion of the doc about it. I find it easy enough to read. I can change it to a separate collection trigger if we want.